### PR TITLE
Enable article level

### PR DESCRIPTION
**Pull request by davidskalinder**
This is the same as https://github.com/MPEDS/mpeds-coder/pull/29, which I managed to auto-delete while juggling branches.  I'm going to paste the substance of that thread into this one here:

> Okay, I've now split the article-level functionality into two hopefully much cleaner PRs.  This one should only enable the functionality to sit in the background with no visibility for users.
>
>It does require a database migration however, namely, the new `coder_article_annotation` table; running `init_db()` should create this successfully (even, I believe, from a live database, although I haven't confirmed this recently).
>
>Other than that, I've copied the `event-creator.html` and `event-creator-block.html` templates from the campus protests project, and uncommented the lines of the controller that display the (event-level) article description in the center panel for events without an event description.
>
>So when I test this on a migrated database, everything looks and acts like it did back in the pre-COVID parts of '19 as far as I can see.  But of course the article-level functionality is waiting under the hood.
>
>Tl;dr: don't accept this PR until database migration is sorted, and after that make sure the PR is tested on a non-critical deployment first; but it should enable all the back-end stuff to code at article-level without actually activating any of it.

### CODE REVIEW

**Comment by alexhanna** [comment]
This is similar to #30, right? But it doesn't break the things I've noted that it will break?

**Comment by davidskalinder** [comment]
The PRs rain like matrix-bullet shells!

> This is similar to #30, right? But it doesn't break the things I've noted that it will break?

Yep, should be.  See the note above about the DB migration though.

**Comment by alexhanna** [comment]
Yeah, shouldn't be an issue. Remind me where the DB migration commands are? Sorry...

**Comment by davidskalinder** [comment]
> Yeah, shouldn't be an issue. Remind me where the DB migration commands are? Sorry...

Can you see them upthread?  I'll quote it just in case:



> This is the same as #29, which I managed to auto-delete while juggling branches. I'm going to paste the substance of that thread into this one here:
> 
> > Okay, I've now split the article-level functionality into two hopefully much cleaner PRs.  This one should only enable the functionality to sit in the background with no visibility for users.
> > It does require a database migration however, namely, the new `coder_article_annotation` table; running `init_db()` should create this successfully (even, I believe, from a live database, although I haven't confirmed this recently).
> > Other than that, I've copied the `event-creator.html` and `event-creator-block.html` templates from the campus protests project, and uncommented the lines of the controller that display the (event-level) article description in the center panel for events without an event description.
> > So when I test this on a migrated database, everything looks and acts like it did back in the pre-COVID parts of '19 as far as I can see.  But of course the article-level functionality is waiting under the hood.
> > Tl;dr: don't accept this PR until database migration is sorted, and after that make sure the PR is tested on a non-critical deployment first; but it should enable all the back-end stuff to code at article-level without actually activating any of it.

**Comment by davidskalinder** [comment]
Oh wait hang on I read that wrong.  Let me find the actual commands

**Comment by davidskalinder** [comment]
Ah, I think this is the key:

> It does require a database migration however, namely, the new `coder_article_annotation` table; running `init_db()` should create this successfully (even, I believe, from a live database, although I haven't confirmed this recently).

I think that'll work since it's  whole new table.  But best to test it first on a dev DB obvs.

**Comment by alexhanna** [comment]
Ah, okay. Will give it a shot on the dev server then. I'll merge this into master.
